integration_tests/redis/redis.py

```diff
@@ -0,0 +1,115 @@
+from unittest import TestCase
+from unittest.mock import patch
+
+from feats.redis import FeatureStream
+from feats.redis import RedisClient
+from feats.redis import StreamIterator
+from feats.redis import NotConnectedError
+
```

Comment:
A nice test to have here would be one with two clients connected to the same redis instance -- the clients should be able to interact with the changes from the other.  A single host redis instance is strongly consistent, so read after write should work fine.

Reply:
I added a few at the bottom, let me know if you think of any more test behaviors you think it would be useful to test there and I can add them.